From 7165b2fc734279fe5477922fc1d27c34f45af1ba Mon Sep 17 00:00:00 2001
From: Chih-Hung Hsieh <chh@google.com>
Date: Tue, 2 Jan 2018 10:04:27 -0800
Subject: [PATCH 11/12] msm8974: Add -Wno-error to compile with global -Werror.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Bug: 66996870
Test: normal builds.
Change-Id: I778435f9370080a95a0dfdaa88fb07d7902874c5
Keep msm8974 in sync with the rest, post removal
Signed-off-by: D. Andrei Măceș <andrei@unlegacy-android.org>
---
 msm8974/core/Android.mk                     | 3 ++-
 msm8974/loc_api/libloc_api_50001/Android.mk | 6 ++++--
 msm8974/utils/Android.mk                    | 3 ++-
 3 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/msm8974/core/Android.mk b/msm8974/core/Android.mk
index b7fb8c3..c0158f1 100644
--- a/msm8974/core/Android.mk
+++ b/msm8974/core/Android.mk
@@ -29,7 +29,8 @@ LOCAL_SRC_FILES += \
 LOCAL_CFLAGS += \
      -fno-short-enums \
      -D_ANDROID_ \
-     -Wno-unused-parameter
+     -Wno-unused-parameter \
+     -Wno-error \
 
 LOCAL_C_INCLUDES:= \
     $(TARGET_OUT_HEADERS)/gps.utils
diff --git a/msm8974/loc_api/libloc_api_50001/Android.mk b/msm8974/loc_api/libloc_api_50001/Android.mk
index 55e83cc..0bfbaef 100644
--- a/msm8974/loc_api/libloc_api_50001/Android.mk
+++ b/msm8974/loc_api/libloc_api_50001/Android.mk
@@ -38,7 +38,8 @@ LOCAL_SRC_FILES += \
 LOCAL_CFLAGS += \
      -fno-short-enums \
      -D_ANDROID_ \
-     -Wno-unused-parameter
+     -Wno-unused-parameter \
+     -Wno-error \
 
 LOCAL_C_INCLUDES:= \
     $(TARGET_OUT_HEADERS)/gps.utils \
@@ -82,7 +83,8 @@ LOCAL_SRC_FILES += \
 
 LOCAL_CFLAGS += \
     -fno-short-enums \
-    -D_ANDROID_
+    -D_ANDROID_ \
+    -Wno-error \
 
 ifeq ($(TARGET_USES_QCOM_BSP), true)
 LOCAL_CFLAGS += -DTARGET_USES_QCOM_BSP
diff --git a/msm8974/utils/Android.mk b/msm8974/utils/Android.mk
index 1135da9..cd4237b 100644
--- a/msm8974/utils/Android.mk
+++ b/msm8974/utils/Android.mk
@@ -24,7 +24,8 @@ LOCAL_SRC_FILES += \
 
 LOCAL_CFLAGS += \
      -fno-short-enums \
-     -D_ANDROID_
+     -D_ANDROID_ \
+     -Wno-error \
 
 ifeq ($(TARGET_BUILD_VARIANT),user)
    LOCAL_CFLAGS += -DTARGET_BUILD_VARIANT_USER
-- 
2.17.1

